util.rb

```diff
@@ -0,0 +1,21 @@
+module Utilities
+  def save_student(id, age, name, has_parent_permission)
+    File.open('persons.json', 'a') do |f|
+      f.write "{\"id\": \"#{id}\", \"age\": \"#{age}\", \"has_parent_permission\": \"#{has_parent_permission}\"," \
+              "\"name\": \"#{name}\", \"type\": \"student\"} \n"
+    end
+  end
+
+  def save_teacher(id, age, name, specialization)
+    File.open('persons.json', 'a') do |f|
+      f.write "{\"id\": \"#{id}\", \"age\": \"#{age}\", \"specialization\": \"#{specialization}\",  " \
+              "\"name\": \"#{name}\", \"type\": \"teacher\"} \n"
+    end
+  end
+
+  def save_rental(date, person_id, book_title)
+    File.open('rentals.json', 'a') do |f|
+      f.write "{\"date\": \"#{date}\", \"person_id\": \"#{person_id}\", \"book_title\": \"#{book_title}\"} \n"
+    end
+  end
+end
```

Comment:
- [ ] This app appears to save data after entry of details for each section. In this activity, we actually want the app to only persist data into _json_ files during exit of the app, thus, after _option 7_ has been selected. Let's improve upon this aspect